api/validation.go

```diff
@@ -469,13 +469,20 @@ func (e *EverestServer) validateBackupStoragesFor(ctx context.Context, databaseC
 		}
 	}
 
-	if databaseCluster.Spec.Backup.Pitr != nil && databaseCluster.Spec.Backup.Pitr.Enabled {
-		storage, err := e.validateBackupStoragesAccess(ctx, databaseCluster.Spec.Backup.Pitr.BackupStorageName)
+	if databaseCluster.Spec.Backup.Pitr == nil || !databaseCluster.Spec.Backup.Pitr.Enabled {
+		return nil
+	}
+
+	if databaseCluster.Spec.Engine.Type == DatabaseClusterSpecEngineType(everestv1alpha1.DatabaseEnginePXC) {
+		if databaseCluster.Spec.Backup.Pitr.BackupStorageName == nil {
```

Comment:
I know this is kinda of covered by `validateBackupStoragesAccess` but maybe it would be better to return `errPitrNoBackupStorageName` if the string is empty. WDYT?
```suggestion
		if databaseCluster.Spec.Backup.Pitr.BackupStorageName == nil || *databaseCluster.Spec.Backup.Pitr.BackupStorageName == "" {
```